components/server/src/server.ts

```diff
@@ -319,8 +319,13 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
             help: 'Currently served websocket connections',
             labelNames: ["clientType"],
         });
-        this.websocketConnectionHandler.onConnectionCreated((_, req) => gauge.inc({ clientType: WebsocketClientType.getClientType(req) || "undefined" }));
-        this.websocketConnectionHandler.onConnectionClosed((_, req) => gauge.dec({ clientType: WebsocketClientType.getClientType(req) || "undefined" }));
+        this.websocketConnectionHandler.onConnectionCreated((_, req) => {
+            gauge.inc({ clientType: WebsocketClientType.getClientType(req) || "undefined" })
+            log.info(`new connection for ${WebsocketClientType.getClientType(req)} full headers: ${JSON.stringify(req.headers)}` )
```

Comment:
@iQQBot  This is fine for testing, but I'd rather not merge this change. I have another PR waiting for review that adds traces for websocket dis-/connects; will add the headers there.